Header header
string txt1
string txt2
float64 val1
float64 val2
